Makefile

```diff
@@ -33,12 +33,15 @@ pkg/generated/openapi/%/zz_generated.openapi.go: go.mod go.sum
 	    -o ./ \
 	    -r /dev/null
 
+.PHONY: generate
+generate: $(generated_files)
```

Comment:
nit: to be consistent with the rest of the makefile it would be better to stick to update/verify naming. I would suggest update-generated and verify-generated here.

Reply:
Done